UsbAccess/UsbAccess.cpp

```diff
@@ -25,6 +25,8 @@ const string WPEFramework::Plugin::UsbAccess::METHOD_GET_AVAILABLE_FIRMWARE_FILE
 const string WPEFramework::Plugin::UsbAccess::METHOD_GET_MOUNTED = "getMounted";
 const string WPEFramework::Plugin::UsbAccess::METHOD_UPDATE_FIRMWARE = "updateFirmware";
 const string WPEFramework::Plugin::UsbAccess::METHOD_ARCHIVE_LOGS = "ArchiveLogs";
+const string WPEFramework::Plugin::UsbAccess::METHOD_GET_USB_SIZE = "getUSBSize";
```

Comment:
We need to also change the JSON API Spec UsbAccess.json to add these 2 APIs. Please provide sufficient description for the API and parameters since this goes into the wiki doc at https://rdkcentral.github.io/rdkservices/#/api/UsbAccessPlugin